packages/frontend/src/navigation/AuthenticatedRoutes.tsx

```diff
@@ -78,10 +81,14 @@ const Routes = (): JSX.Element => {
         <MyPraisePage />
       </Route>
 
-      <AuthRoute roles={[ROLE_ADMIN]} path={'/pool'}>
+      <AuthRoute roles={[ROLE_ADMIN]} exact path={'/pool'}>
         <UsersPage />
       </AuthRoute>
 
+      <AuthRoute roles={[ROLE_ADMIN]} path={'/pool/:userId'}>
+        <UserDetailsPage />
+      </AuthRoute>
+
       <Route exact path={'/periods'}>
```

Comment:
Let's change the URL for the user page from `/pool/*` to `/users/` to reflect that its now a general-purpose user-management page, not just for the quantifier users.

Reply:
Agree